Somewhere wrong (old) names are used in accessing the preferences structure.

This is just a remainder for the upstream!!

Antonio Ospite <ospite@studenti.unina.it>

diff -pruN uae-0.8.25/src/tui.c uae-0.8.25_patched/src/tui.c
--- uae-0.8.25/src/tui.c	2006-06-07 17:45:55.000000000 +0200
+++ uae-0.8.25_patched/src/tui.c	2006-06-07 17:55:39.000000000 +0200
@@ -606,7 +606,7 @@ static void SoundOptions (void)
 		currprefs.sound_freq = atoi (tmp);
 	    break;
 	 case 5:
-	    currprefs.stereo = (currprefs.stereo + 1) % 3;
+	    currprefs.sound_stereo = (currprefs.sound_stereo + 1) % 3;
 	    break;
 	}
     }
